main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"game/board"
+	"strings"
+)
+const size = 20
+const MaxRound = 25
+
+func validInput(pick string, options []string) bool {
+	for _, val := range options {
+		if pick == val { return true }
+	}
+	return false
+}
+
+func getNumber (options []string) string {
+	var pick string
+	for !validInput(pick, options){
```

Comment:
When you initialized the pick in line 19, it was set to empty string ("")
What is the point of iterate all the options list and see if the pick is equals to one of them? 

Does the point is to see if an option is set to empty string? 
If so, the method should be more explicit to this goal, and the pick option is not relevant

Reply:
the method checks if the pick is one of the possible options.
how should I check it?